strapi/strapi
# Initialize bulk publish v2 modal

### What does it do?

Initiates a modal to prompt the user to review the selected items before publishing. The modal has its own selected items, which are not in sync with the selected items of the list view.

⚠️ what it does NOT do: ⚠️

- Show the validation status of each item
- Provide a refresh button
- Actually publish the proper items (only the selected ones)

All of the above will be in other PRs, this one is just to get the project up and going.

### How to test it?

- Selected items in a content type with draft & publish enabled
- Click on publish
- Instead of the validation dialog, you should now get the confirmation modal
- The sorting of items in the list view should be respected
- You can select and unselect items in the modal
- The "X entries selected" text should update
